rules_building_block/execution_github_new_event_action_for_pat.toml

```diff
@@ -0,0 +1,60 @@
+[metadata]
+creation_date = "2023/10/11"
+integration = ["github"]
+maturity = "production"
+min_stack_comments = "New fields added to GitHub Integration"
+min_stack_version = "8.8.0"
+updated_date = "2023/12/14"
+bypass_bbr_timing = true
+
+[rule]
+author = ["Elastic"]
+building_block_type = "default"
+description = """
+Detects a new event.action for a GitHub PAT not seen in the last 14 days.
+"""
+from = "now-9m"
+index = ["logs-github.audit-*"]
+language = "kuery"
+license = "Elastic License v2"
+name = "New event.action For A GitHub PAT"
+risk_score = 21
+rule_id = "ce08b55a-f67d-4804-92b5-617b0fe5a5b5"
+severity = "low"
+tags = ["Domain: Cloud",
+        "Use Case: Threat Detection",
+        "Use Case: UEBA",        
+        "Tactic: Execution",
+        "Rule Type: BBR",
+        "Data Source: Github"
+        ]
+timestamp_override = "event.ingested"
+type = "new_terms"
+
+query = '''
+event.dataset:"github.audit" and event.category:"configuration" and
+event.action:* and github.hashed_token:* and 
+github.programmatic_access_type:("OAuth access token" or "Fine-grained personal access token")
```

Comment:
Any chance we can ignore CI or system jobs based on `user.*` fields? This would ignore newly established CI workflows that use a new or existing PAT token.

Reply:
I'm not sure right now but will keep an eye out in telemetry for those tuning opportunities